CMakeLists.txt

```diff
@@ -31,14 +31,21 @@ include(GNUInstallDirs)
 if(BEMAN_EXEMPLAR_BUILD_TESTS)
     # Fetch GoogleTest
     FetchContent_Declare(
-        GTest
+        googletest
+        FETCHCONTENT_TRY_FIND_PACKAGE_MODE
+        ALWAYS
         GIT_REPOSITORY https://github.com/google/googletest.git
-        GIT_TAG
-            6910c9d9165801d8827d628cb72eb7ea9dd538c5 # release-1.16.0
+        GIT_TAG v1.16.0
```

Comment:
I think this was already discussed and we prefer using plain commit id.

https://github.com/bemanproject/exemplar/pull/128#pullrequestreview-2649347980

Please wait for a check from @camio , as this reverts part of the diff in #128 .

 Also, I think we had the block() and we removed it in https://github.com/bemanproject/exemplar/commit/0f082a5ad5cdf796fc77481c73fb2b7383259a5b. @wusatosi , can you take a look?

Reply:
Removing `block()` is a requested change, discussed here: #61

Reply:
regarding the tag, use commit hash maybe preferred as best practice:

As I quoted in the [review comment](https://github.com/bemanproject/exemplar/pull/128#pullrequestreview-2649347980):
> [source](https://cmake.org/cmake/help/latest/module/FetchContent.html):
> Where contents are being fetched from a remote location and you do not control that server, it is advisable to use a hash for GIT_TAG rather than a branch or tag name. A commit hash is more secure and helps to confirm that the downloaded contents are what you expected.

I personally don't see the benefit of commit hash when the tag is specified down to patches. But I will leave the decision to @camio @bretbrownjr .